src/pages/gestion/listado.ts

```diff
@@ -33,6 +38,12 @@ export class ListadoComponent implements OnInit {
     async cargarDatos() {
         let consulta = await this.datosGestion.areasPorZona(this.id)
         if (consulta.length) {
+            console.log(consulta)
+            if(this.authProvider.esDirector >= 0 ) {
```

Comment:
if(this.authProvider.esDirector >= 0 ) {
               let temporal = consulta.filter(x => Number(x.IdArea) === this.user.idArea)

Se repite aca y en listadoDetalle. Sería bueno refactorizar y  rehusar código.